Prune alffd fluff

Allfd opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

<!-- NOTE: This format is NOT REQUIRED for things like runtime fixes, code fixes and optimizations. In those instances you should try to give a description of what is being fixed or optimized but are not required to fill out the complete changelog. -->
<!-- Adjusting things like armor or weapon values for balance, while they may be 'fixes' in their own way, are not considered code fixes as described above and require the use of the Pull Request format below.-->


## About The Pull Request

This pull requests guts most of the items added in my initial fluff PR, while I argue that my fluff was by far not the most balance altering, it would come up in arguments.  The purpose of this PR is to remove "fluff" based arguments entirely from the minds of neutral third-part observers to dev-chat arguments.

Everything not a base level organ replacement has been removed.

## Why It's Good For The Game

<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

In that the original list did not alter anything, removals from that list also should not alter anything.

## Pre-Merge Checklist
<!-- Don't bother filling these in while creating your Pull Request, just click the checkboxes after the Pull Request is opened and you are redirected to the page. -->
- [X] You tested this on a local server.
- [X] This code did not runtime during testing.
- [X] You documented all of your changes.
<!-- Neither the compiler nor workflow checks are perfect at detecting runtimes and errors. It is important to test your code/feature/fix locally. -->


## Changelog
<!-- This is mostly optional for small Pull Requests, but if you value being credited for your work in-game be sure to fill it out. To opt-out, remove everything below including the start and end :cl: brackets. -->

<!-- If your Pull Request includes a minor single line variable edit, probably do not fill out this changelog. -->
<!-- However, if your pull request includes massive or immediately noticeable changes, briefly describe those changes in some way in this changelog. -->

:cl:
del: Removed all non base level organs from alffd's fluff list.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

ProbablyCarl commented:
I don't necessarily see the reason to gut your own fluff, with the statements of a few people who, mind, don't even play, being the driving factor.
I can merge this if you'd like, but I don't feel it proper, personally.

Allfd commented:
I was unaware of the lurker status, the initial purpose of this was as a hold over while I worked on synth changes.  Would it be acceptable to drop the reviver implant and medihud, then remove the Enclave checks?  Those implants really serve no purpose other then apparently discord arguments?

Will do that change now in another commit for clarity.

Allfd commented:
Health-hud and reviver implant yeeted, they were only there to make my character more suspicious to the Brotherhood, something I probably don't need help with.   Those could have been used for PVP on a non-scientist role, with them gone, I also removed those checks and simplified things. 

I also customized the previous vestigial brain implant to cause dumb side-effects when EMPed.
